### ISSUE TITLE
💬 Discussion | Alternative to Facebook Events

### ISSUE THREAD

**Issue by pdjpdjpdj** [privacytoolsIO/privacytools.io#754]
Facebook is ever present with it's 2billion plus user base. Yet it's hard to just ditch it altogether and one of the factors is of course the network effect and specifically for Events. Do you guys see/know any viable alternatives?

It should be:
- easy to use
- easy to migrate to (start with scraped public Fb events?)
- contain important event features (attendance, location, discussion, ...)
- privacy respecting (ideally with strong cryptographic guarantees)

**Comment by beerisgood** [comment]
Fediverse

**Comment by PrivSecurity** [comment]
I think Hubzilla or Friendica are the two Fediverse platforms that are closest to what you are asking for. Correct me if I am wrong. Here is some info regarding the differences between the two:

https://github.com/friendica/friendica/issues/2894

**Comment by pdjpdjpdj** [comment]
Thanks both for the info. I haven't tried those will give it a shot. Are they interconnected Hubzilla and Friendica? Can a bridge be made for Mastodon?

**Comment by PrivSecurity** [comment]
If I remember correctly both hubzilla and friendica  can have friends on and post to Mastodon as well as other platforms on the free Social Web. If you read the link I posted I think it says there that those two are the only platforms that can you connect to both fediverse and Federated or whatever they're called those two parts of the free Social Web.

**Comment by Vanecx** [comment]
I think what you're looking is already under construction thanks to Framasoft : https://github.com/framasoft/mobilizon

**Comment by strypey** [comment]
It's a bit dated now, but Sean Tilley's 'We Distribute' article ['A Quick Guide to the Free Network'](https://wedistribute.org/2017/09/a-quick-guide-to-the-free-network/) is still a good overview of the history of the various federated  social network/ social media apps, which ones have inter-operated with each other, and over which protocols (OStatus/ Diaspora/ Zot/ ActivityPub). These days, AFAIK the only apps that are not either federating over ActivityPub or planning to (or defunct), are Diaspora (supports only its own protocol) and Zap (supports only Zot). The terminology is still evolving, but because of this rapid convergence on AP (as predicted in Sean's article), for fediverse.party we have decided (somewhat arbitrarily) to use "the fediverse" refer to the whole spaghetti soup of federated web apps. This is also how it's used in the [fediverse article on Wikipedia](https://en.wikipedia.org/wiki/Fediverse), which features a useful table of which apps support which protocols (although somebody has made a mess of the article text).

Coming back to events, some of the fediverse apps already have some support for them, and there are a number of projects in the works that intend to federate events over AP, including Mobilizon, GetTogether, FedEvent, and possibly NextCloud. There seems to be some need for coordination between the developers of the various projects to [standardize some extensions to AP that will be needed for events to federate smoothly](https://socialhub.network/t/federated-events/226) across instances of different apps, and maybe develop federated events libraries in each language (PHP, Ruby, Python etc) that apps can use, rather than all rolling their own AP events support from scratch.

In summary, it seems like we're on the cusp of some really exciting developments in the federated events space, but help is needed to get all the moving parts working in sync.

**Comment by daveloyall** [comment]
@strypey, would you provide a link to FedEvent?  It's easy to find links to [Mobilizon](https://framagit.org/framasoft/mobilizon) and [GetTogether](https://github.com/GetTogetherComm/GetTogether).  Regarding NextCloud, I assume you're referring specifically to [their 'social' application](https://github.com/nextcloud/social), correct?

Thank you!

**Comment by Vanecx** [comment]
FYI, Framasoft just launched a few hours ago its crowdfunding for Mobilizon, you can find it [here](https://joinmobilizon.org/en/), they've already reached 30% of their first goal (20.000€).

**Comment by strypey** [comment]
@daveloyall it was in the third link I posted, a couple of comments down, but here you go:
https://github.com/shiburizu/fedevent

It's labeled as a prototype and I'm not sure if it's still in active development, nothing has been committed to the repo there since the code was dumped over the wall 4 months ago.

**Comment by Mikaela** [comment]
Mobilizon has gotten a test instance out at https://test.mobilizon.org/ .

I think I will close this as we nowadays have the forum and there isn't any concrete suggestion to be litsed to PTIO yet (feel free to open one).

**Comment by Mikaela** [comment]
Reopening and assigning to me in case we want to add a section for this to social media.

\+ I hope Mobilizon can be listed, but we will see.

**Comment by bkil** [comment]
See also: https://github.com/coderbyheart/open-source-meetup-alternatives

**Comment by Mikaela** [comment]
I had forgotten this issue, but Mobilizon has been released and is at version 1.2 or so. I hope it can be considered nowadays, while the pandemic renders events a bit less useful than they usually are.

* https://mobilizon.org/en/ - quick introduction
* https://joinmobilizon.org/en/ - longer information
* https://instances.joinmobilizon.org/instances - where to find instances to register on

**Comment by bkil** [comment]
Yes, Mobilizon had been mentioned multiple times both in this thread and in the linked table. Hopefully as more and more alternatives progress they will eventually become interoperable.

We should probably not put all of our eggs into the same basket.
